saltedge-compliance-connector-java-v2/SDK_CHANGELOG.MD

```diff
@@ -1,5 +1,12 @@
 # Salt Edge Compliance Connector SDK Changelog
 
+## 2.9.0 (2021-11-19)
+### Add:
+- Add `funds_available` in session update callback
+
+### Change:
+- Add check for payment description on nullability, update the parameters for the session update request
```

Comment:
`Add check for payment description on nullability` - this change not related to SDK